docs/tutorials/ble/client.mdx

```diff
@@ -9,27 +9,27 @@ In order for a BLE device to establish a connection to a remote device, the devi
 
 ```toit
 import ble
-import uuid
 
-// Find the address with the service.
-find_with_service device/ble.Device service/uuid.Uuid -> ble.Address:
-  device.scan: | remote_device/ble.RemoteDevice |
-    if remote_device.data.service_classes.contains service:
-      return remote_device.address
+SCAN_DURATION ::= Duration --s=3
+
+find_with_service central/ble.Central service/ble.BleUuid:
+  central.scan --duration=SCAN_DURATION: | device/ble.RemoteScannedDevice |
+    if device.data.service_classes.contains service:
+        return device.address
   throw "no device found"
 ```
 
 With the address available, the connection can be established. In the following example, we use the `find_with_service` function to find a remote device with a battery service and establish a connection to that device.
```

Comment:
With the address available, the connection can be established. In the following example, we use the `find_with_service` function to find a remote device with an advertised battery service and establish a connection to that device. Note that the device might also support services that are not explicitly advertised,

Reply:
done.